### DISCUSSION TITLE
GitHub Education benefits

### DISCUSSION THREAD

**Discussion by a-lgil** [community/community#140028] [GitHub Education]
### Hi everyone,

Hi,

I'm creating this discussion following the suggestion that says "If you have not received your education benefits within 5 days of your approval status please raise a discussion on the GitHub Education community and the Education community team will be able to assist in resolving this for you."

My submission was sent on the 18 of September and approved a day later, after which i got the "Congrats! Your request for @a-lgil was approved and your benefits will be available soon." message, and I've been waiting since then.

Thanks in advance.

**Comment by davevad93** [comment]
Hi @a-lgil ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @a-lgil was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
